ext/oboe_metal/README.md

```diff
@@ -65,3 +65,86 @@ Some inspiring examples here:
 <https://jvns.ca/blog/2016/06/12/a-weird-system-call-process-vm-readv/>
 
 <https://medium.com/@zanker/finding-a-ruby-bug-with-gdb-56d6b321bc86>
+
+## Debug the c-code with core dump
+
+If the core dump is produced, it's easier to check the backtrace with it in gdb
+
+For example, `(core dumped)` indicate the crash file is dumped
+
+```console
+./start.sh: line 37:    65 Segmentation fault      (core dumped) ...
+```
+
+### Prepare
+
+#### 1. Install solarwinds_apm with debug symbol on
+
+```console
+export OBOE_DEBUG=true  # enable debug flag when compiling; and also download *.debug into lib/ when create the gem for ease of use
```

Comment:
Can we have `OBOE_DEBUG=true` install the "RelWithDebInfo" version of liboboe? Then no need to deal with the *.debug symbols file.

Reply:
Updated